src/components/LoadMore/LoadMoreButton.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+// @ts-expect-error
+import { LoadingIndicator } from 'react-file-utils';
```

Comment:
looks like we need to type this lib. @ath92 in my 1on1 with Ben today he mentioned you'll be helping starting next sprint, this might be a good place for you to start

Reply:
Cool, yeah makes sense. I think there was some discussion a while back about whether this should be a separate package, I think it's kinda outdated. But something I can look into when I get started :)

Reply:
@ath92 great, discussed this with Ben as well. Could be a nice way to get going on TS since this library doesn't need to be converted for us to release a TS version of stream-chat-react, so you can take it at your own pace as you ween off the dashboard